hardhat.config.ts

```diff
@@ -129,6 +130,46 @@ task('deploy', 'Deploy contracts').setAction(async (args, hre, runSuper) => {
   return runSuper(args);
 });
 
+task('deploymentsPrettier', 'Adds EOF new line to prevent prettier to change files').setAction(async (args) => {
+  if (!fs.existsSync('./deployments')) {
+    throw new Error('The directory "deployments" was not found.');
+  }
+
+  function getAllFiles(dirPath: string, arrayOfFiles: string[]) {
+    const files = fs.readdirSync(dirPath);
+
+    arrayOfFiles = arrayOfFiles || [];
+
+    files.forEach(function (file) {
```

Comment:
Nit: can this be updated to also use `for (const file of files) {` for consistency